src/Components/WebAssembly/Authentication.Msal/src/Microsoft.Authentication.WebAssembly.Msal.csproj

```diff
@@ -32,7 +32,6 @@
   <ItemGroup>
     <YarnInputs Include="$(YarnWorkingDir)**" Exclude="$(YarnWorkingDir)node_modules\**;$(YarnWorkingDir)*.d.ts;$(YarnWorkingDir)dist\**" />
     <YarnOutputs Include="$(YarnWorkingDir)dist\$(Configuration)\AuthenticationService.js" />
-    <YarnOutputs Include="$(YarnWorkingDir)dist\$(Configuration)\AuthenticationService.js.map" Condition="'$(Configuration)' == 'Debug'" />
```

Comment:
This doesn't seem to be being generated, but `YarnOutputs` is explicitly added to the `UpToDateCheckOutput` below.

Reply:
Yeah, that's what I figured.  I wondered if anyone who knew yarn things knew whether it matters and if so, how to turn it back on instead.

Reply:
@javiercn

Reply:
@dougbu - but the point of this PR is that people *do* build Debug locally in VS, and this makes their experience worse when doing so.

Reply:
Ah, I misunderstood the situation 👍